scripts/gen-service.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+OUTDIR=${1:-build}
+mkdir -p "$OUTDIR"
+
+OUTPUT=$OUTDIR/gluster-exporter.service
+
+ARGS=""
+if [ "$GLUSTER_MGMT" == "glusterd2" ];then
+    ARGS+="--gluster-mgmt=glusterd2"
+fi
+
+cat >"$OUTPUT" <<EOF
+[Unit]
+Description=Gluster Prometheus Exporter
+
+[Service]
+ExecStart=${SBINDIR}/gluster-exporter ${ARGS}
```

Comment:
How do we take care of other command line options like `--config`, `--volinfo` etc with this?

Reply:
`--volinfo` will be removed once the PR https://github.com/gluster/gluster-prometheus/pull/35 merges.

`--config` can be added here once it is merged.

Reply:
Ack